### PR TITLE
Feature/sal/#9-add_dash

### PR DESCRIPTION
**Pull request by SalMireles**
Embedds a dash app into flask for some quick prototyping. This just sets up the app with some test data. A few todo's that I may punt to future PR's are:

(1) Cleaning up the page layout and put behind a user login (flow tbd)
(2) Determine the data that we would like to show on this page
(3) Remove the test data and begin populate these with data from api's we target or from the db we set up

Home page:
image

Dashboard Page:

image

### CODE REVIEW

**Review by misaacs — APPROVED** [review]
I'm not familiar with dash but I ran this branch; works fine. I'm gonna approve it to speed things up & make sense of it down the road. Nice work Sal!

Please add any dash resources that you like; here's one- https://hackersandslackers.com/plotly-dash-with-flask/
Why are numpy & pandas included?

There was a merge conflict over the poetry.lock content hash info; It's awkward editing a generated file but I remove the dev-main hash in favor of the "new" hash; in case of issues: regenerate the .lock from .toml

**Comment by SalMireles** [comment]
I merged the updates from #29. This put the dashboard behind a route that's displayed in the navbar

image

**Comment by SalMireles** [comment]
Recent update: make dash work with bootstrap
image

**Comment by SalMireles** [comment]
now uses real data from coingecko:

image

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,7 +4,9 @@ repos:
   hooks:
   - id: black
     language_version: python3.8
+    args: ["--line-length=80"]
```
Comment (SalMireles) [inline .pre-commit-config.yaml:7]: @misaacs i ran into the issue you mentioned about conflicting line lengths and added these arts to resolve the conflict

**Review by misaacs — APPROVED** [review]

**Review by CloudF1sh — APPROVED** [review]

**Pull request merged**